README.md

```diff
@@ -507,6 +507,18 @@ as key:value.
 When parsing the value, the first colon divides key and value. All other
 colons are ignored.
 
+## Resetting an argument
+
+Existing stored parameters can be reset back to their default value. This
```

Comment:
I'm not sure, but it might be good to better explain what's the default value. Assume that you have installer based on kafo (e.g. foreman-installer) which ships some "default" values in packaged answer file. This reset feature actually resets the value to "system default" right?

Btw, thanks for automatically documenting all new additions!